src/components/ScholarshipDetailCard.jsx

```diff
@@ -1,29 +1,112 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Button, Box, Link, makeStyles, Typography } from '@material-ui/core';
+import {
+  Button,
+  Box,
+  Link,
+  makeStyles,
+  Typography,
+  Chip,
+} from '@material-ui/core';
+import { Share, Send, Info } from '@material-ui/icons';
 import ScholarshipAmount from '../types/ScholarshipAmount';
+import ScholarshipEligibility from '../types/ScholarshipEligibility';
 
-const useStyles = makeStyles(() => ({
+import ScholarshiDetailCardCell from './ScholarshipDetailCardCell';
+
+const useStyles = makeStyles((theme) => ({
+  name: {
+    fontWeight: '500',
```

Comment:
Probably not necessary. The Figma design is not a perfect mock up.

Reply:
> Probably not necessary. The Figma design is not a perfect mock up.

Ah just know this today.